### PR TITLE
fix: cosmwasm1.0 about module

### PR DESCRIPTION
**Pull request by nowooj**

### CODE REVIEW

File: content/en/docs/develop/core-modules/wasm.md [hunk 1]
```diff
@@ -142,9 +156,18 @@ Uploads new code to the blockchain and results in a new code ID, if successful.
 
 ```go
 type MsgStoreCode struct {
-	Sender sdk.AccAddress `json:"sender" yaml:"sender"`
+	// Sender is the that actor that signed the messages
```
Comment (jhlee-young) [inline content/en/docs/develop/core-modules/wasm.md]: plz remove all `that` prior to `actor`

Comment (nowooj) [reply content/en/docs/develop/core-modules/wasm.md]: https://github.com/xpladev/docs/pull/10/commits/07c7f7870d2891a3dbc8059de5f50b835300f417

**Review by jhlee-young — APPROVED** [review]
LGTM

**Review by caramis — APPROVED** [review]
LGTM

**Pull request merged**